gistUpload.js

```diff
@@ -1,27 +1,34 @@
 const axios = require('axios');
 const fileSystem = require('fs');
+require('dotenv').config();
 
 const API_URL = 'https://api.github.com/gists';
 let finalString = '';
 const PROG_BAR = '=';
 const BAR_LENGTH = 30;
 const HANG_PERCENT = 0.25;
 
+
 /** This is a description of the uploadFile function
  * @param {string} contents - Contents of file from readFile
  * @param {string} gist - Name of gist supplied by user
  */
 function uploadFile(contents, gist) {
   const post = { [gist]: { content: contents } };
```

Comment:
Try to use variable names/definitions to make your code more concise. To clean this up you could say
` function uploadFile(content, gist) {`
`const files = { files: { [gist]: { content } } }`
`return axios.post(API_URL, files)`

Reply:
I am able to cut line 17 out, but I have to use the full 
````
return axios(method..., url, auth..., data..)
````
in order to correctly authenticate to github's API.